requirements/runtime.in

```diff
@@ -1,4 +1,4 @@
-twine
+twine ~= 6.0
```

Comment:
I just realized that we didn't really need to cap this below v7 since this input file is not used w/o the constraints anyway: https://github.com/pypa/gh-action-pypi-publish/commit/67339c736fd9354cd4f8cb0b744f2b82a74b5c70